float64 desired_hdg

---
bool success
